src/unittest/python/plugins/python/cram_plugin_tests.py

```diff
@@ -31,6 +45,8 @@
 
 
 class CramPluginTests(unittest.TestCase):
+
```

Comment:
This is a mock test that doesn't execute anything. No reason to disable those tests.